core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,50 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
```

Comment:
Although the aim of this block of CSS is to fix the exploration player, I believe it applies to all pages. It's very important, when making a PR, to not only ensure that it fixes the issue you're fixing, but also to ensure that it does not have side effects.